packages/react-native-editor/metro.config.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
 				inlineRequires: false,
 			},
 		} ),
+		unstable_allowRequireContext: true, // Used for optional setup configuration.
```

Comment:
We'd need to keep an eye on this unstable parameter when upgrading Metro (probably in the next React Native upgrade effort), in case it produces a failure in loading the local setup.